### PR TITLE
Convert Card Component

### PR DESCRIPTION
**Pull request by abeprincec**

### CODE REVIEW

**Comment by abeprincec** [comment]
related to #27

**Comment by abeprincec** [comment]
Not sure how we will do styling for now. Just used inline css for now

File: src/components/card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+interface Props {
+  children?: React.ReactNode
+}
+
+function Card({ children }: Props) {
+  return (
+    <div
+      className="card"
+      style={{
```
Comment (YPAzevedo) [inline src/components/card/index.tsx:11]: Are we doing everything inline while we dont make a decision on styling?

Comment (abeprincec) [reply src/components/card/index.tsx:11]: I am not sure I have just used inline since it would be easy to convert later.

**Comment by YPAzevedo** [comment]
We added https://github.com/OpenSourceRaidGuild/website/pull/32 to the project, can you update the PR to use emotion with the styled component approach?

**Review by nobrayner — APPROVED** [review]

**Pull request merged**